client/android/build.gradle

```diff
@@ -173,18 +174,6 @@ android {
        }
    }
 
-//   externalNativeBuild {
```

Comment:
Please also cleanup cmake sections above - https://github.com/amnezia-vpn/amnezia-client/pull/252/files#diff-61c7bfb2e67d762262c9bd49ba0821b2ff9c445d7c3f2f7dcf1e8930553e56f9L166

Please also cleanup cpp folder and remove unnecessary submodules and folders

Reply:
We use this folder and submodules in [3rd-build-ga](https://github.com/amnezia-vpn/3rd-build-ga). We pull amnezia project and build binaries with CMake. 

We can only remove openvpn, mbedtls because of openvpn3 and openssl usage.